.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules
```

Comment:
- [ ] Great Job adding test.md to .gitignore. However please kindly add a forward slash `/` in front of  node_modules. This indicated that it's a folder.
### EXAMPLE
`node_modules/`

Reply:
Kindly, implement the above changes suggested by the previous reviewer.

Reply:
- [ ] @ShuduK good job, but I can see you have not yet implemented the change requested, the `node_modules` is a directory so ignoring it you need to add a front-slash to be like this `node_modules/`.